### ISSUE TITLE
PiVPN installer immediately crashes back to terminal on Debian Stretch

### ISSUE THREAD

**Issue by Fayyaadh** [pivpn/pivpn#516]
Hi there

Fresh installation of Debian Stretch here and the PiVPN installer keeps crashing back to the terminal.

```
pi@raspberrypi:~ $ curl -L https://install.pivpn.io | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   178  100   178    0     0     80      0  0:00:02  0:00:02 --:--:--    80
100 55909  100 55909    0     0  16764      0  0:00:03  0:00:03 --:--:-- 51011
:::
::: sudo will be used for the install.
::: Verifying free disk space...
:::
::: Checking apt-get for upgraded packages.... done!
:::
::: Your system is up to date! Continuing with PiVPN installation...
pi@raspberrypi:~ $
```

## Have you searched for similar issues and solutions? 
yes, looks the same as issue #399 to me.
  
Some assistance please?

**Comment by cfcolaco** [comment]
Have you checked what I mention in this comment?? 

https://github.com/pivpn/pivpn/issues/399#issuecomment-358428349

**Comment by Fayyaadh** [comment]
I briefly skimmed over that but didn't actually try that.

BUT

I initially was trying to get it installed over SSH but that kept crapping out. So I connected my pi directly to a monitor and keyboard and it seems to work then.  NOTE. my stupid keyboard attached to the raspberry pi doesn't have a "|' key so I basically redirected the contents of the curl to a file I called pivpn.sh and then did sudo ./pivpn.sh to kick off the installer.

That seems to have worked. So it's only over SSH that the installer is crashing back to bash terminal.
